database/migrations/20220110072026_applicants.js

```diff
@@ -0,0 +1,34 @@
+exports.up = async function (knex) {
+  await knex.raw(`create extension if not exists "uuid-ossp"`);
+  return await knex.schema.createTable("applicants", function (table) {
+    table
+      .uuid("uuid")
+      .notNullable()
+      .primary()
+      .defaultTo(knex.raw("uuid_generate_v4()"));
+    table.string("user_uuid", 255).notNullable();
+    table.string("full_name", 255).notNullable();
```

Comment:
@ShubhamJoshi-1994 @rajeshn95 Many of these columns are not needed, can be pushed into one big jsonb. Think like noSQL.

What is the main purpose of "hire", to find jobs, post jobs, filter jobs based on some criteria. Even if "profile" itself is an important aspect, not all details. Some might have larger profiles, some might have smaller than what columns we have. We can't insist that these would be columns.

Reply:
Instead of calling these "applicants" - because they didn't apply to any job. They are just "seekers" at this point, seeking a job. 

It can be like:
- There are jobs
- There are seekers
- Seekers apply for job and become applicant for the job
- A Seeker can apply to multiple jobs
- A seeker would filter jobs
- System can intelligently find matching jobs or notify on new jobs (future)